cmd/ddev/cmd/debug-dockercheck.go

```diff
@@ -41,18 +41,21 @@ var DebugDockercheckCmd = &cobra.Command{
 		case "docker desktop":
 			dockerutil.IsDockerDesktop()
 		}
-		util.Success("Using Docker context: %s (%s)", dockerutil.DockerContext, dockerutil.DockerHost)
-		util.Success("docker-compose: %s", versionInfo["docker-compose"])
 
-		dockerHost := os.Getenv("DOCKER_HOST")
-		if dockerHost != "" {
-			util.Success("Using DOCKER_HOST=%s", dockerHost)
-		}
+		util.Success("Using Docker context: %s", dockerutil.DockerContext)
 		dockerContext := os.Getenv("DOCKER_CONTEXT")
 		if dockerContext != "" {
-			util.Success("Using DOCKER_CONTEXT=%s", dockerContext)
+			util.Success("From DOCKER_CONTEXT=%s", dockerContext)
+		}
+
+		util.Success("Using Docker host: %s", dockerutil.DockerHost)
+		dockerHost := os.Getenv("DOCKER_HOST")
+		if dockerHost != "" {
+			util.Success("From DOCKER_HOST=%s", dockerHost)
```

Comment:
I didn't remove this just because we need more debugging information in the `ddev debug test`. If they have set some env or not, etc.